### PR TITLE
Arbitrary call

### PR DESCRIPTION
**Pull request by Yhtiyar**

### CODE REVIEW

**Comment by Yhtiyar** [comment]
@ndkirillov  I forgot to add `.md` description :(
Will add today

**Comment by Yhtiyar** [comment]
fixed, ready for review

File: docs/arbitrary_call.md
Comment (ndkirillov) [inline docs/arbitrary_call.md:1]: Lines 1-7 and 13-15 are from the other detector. Please change to the PRs' subject detectors.

**Review by ndkirillov — CHANGES_REQUESTED** [review]
Some changes are needed in the documentation file. Please check the related comment.

File: slither_pess/detectors/arbitrary_call.py
[the remark this answers is not in the document]
Comment (Yhtiyar) [reply slither_pess/detectors/arbitrary_call.py:1]: added

File: docs/arbitrary_call.md
Comment (Yhtiyar) [reply docs/arbitrary_call.md:1]: fixed

**Review by ndkirillov — APPROVED** [review]
All good now

**Pull request merged**